SPECS/libcap-ng/libcap-ng.spec

```diff
@@ -41,6 +47,15 @@ The libraries and header files needed for libcap_ng development.
 %make_install
 find %{buildroot} -type f -name "*.la" -delete -print
 
+
+# Remove a couple things so they don't get picked up
+rm -f $RPM_BUILD_ROOT%{_libdir}/libcap-ng.la
+rm -f $RPM_BUILD_ROOT%{_libdir}/libcap-ng.a
+rm -f $RPM_BUILD_ROOT%{_libdir}/libdrop_ambient.la
+rm -f $RPM_BUILD_ROOT%{_libdir}/libdrop_ambient.a
+rm -f $RPM_BUILD_ROOT%{_libdir}/python%{python3_version}/site-packages/_capng.a
+rm -f $RPM_BUILD_ROOT%{_libdir}/python%{python3_version}/site-packages/_capng.la
+
```

Comment:
The `find` command above already removes the `*.la` files. Instead of adding these lines (along with the outdated `RPM_BUILD_ROOT`), could you update the `find` command instead? But before that: see my comment about `*.a` files below.